jest.config.js

```diff
@@ -128,9 +128,14 @@ const config = {
         "<rootDir>/enterprise/frontend/src/embedding-sdk/**/*.unit.spec.{js,jsx,ts,tsx}",
       ],
 
+      setupFiles: [
+        ...baseConfig.setupFiles,
+        "<rootDir>/enterprise/frontend/src/embedding-sdk/jest/setup-env.js",
```

Comment:
This setup is a part of [SDK tests](https://github.com/metabase/metabase/pull/60049/files#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1R128) only.